tools/gulp/docs/index.js

```diff
@@ -185,7 +185,8 @@ module.exports = (gulp, shared) => {
     );
 
     const packages = shared.packages.map(pkg => `packages/${pkg}/src/`);
-    return kss.traverse(packages)
+    const mask = '*.css|*.less|*.sass|*.scss|*.jsx';
```

Comment:
We also have `.jsx` files for examples and tests (ie. `Choice.example.jsx`, `Choice.test.jsx`), which this will traverse over. Not sure if that will have any noticeable performance impact, but if we could ignore those, that'd be ideal.